java/client/src/org/openqa/selenium/remote/HttpCommandExecutor.java

```diff
@@ -496,15 +496,15 @@ private Response createResponse(HttpResponse httpResponse, HttpContext context,
     return response;
   }
 
-  private static CommandInfo get(String url) {
+  protected static CommandInfo get(String url) {
```

Comment:
I suggest we revert all changes in this file. They are not necessary. Other classes that already create instances of CommandInfo outside of this class simply use CommandInfo constructor(s), we could keep doing the same here.